include/deal.II/particles/particle_handler.h

```diff
@@ -847,17 +849,11 @@ namespace Particles
     std::unique_ptr<PropertyPool<dim, spacedim>> property_pool;
 
     /**
-     * Set of particles currently living in the local domain, organized by
-     * the level/index of the cell they are in.
+     * Set of particles currently living in the local domain including ghost
+     * cells , organized by the level/index of the cell they are in.
```

Comment:
Another leftover in the documentation:

```suggestion
     * cells , organized by the active cell index of the cell they are in.
```